docs/translating.md

```diff
@@ -17,7 +17,6 @@
 2. After registering check if you got an email to verify your account and click the link (if there is none head to step 1.4)
 3. Log into weblate
 4. Head to https://translate.element.io/accounts/profile/ and select the languages you know and maybe another language you know too.
-6. Head to https://translate.element.io/accounts/profile/#subscriptions and select Element Web as Project
```

Comment:
I might be wrong, but I would assume that subsciptions have been renamed to notifications.
The description above is not very clear, but I can only assume that this is a step for you to be notified about new strings added to the project

https://translate.element.io/accounts/profile/#notifications